tests/FunctionsFrameworkTest.php

```diff
@@ -56,6 +59,20 @@ public function testRegisterAndRetrieveCloudEventFunction(): void
         );
     }
 
+    public function testRegisterAndRetrieveTypedFunction(): void
+    {
+        $fn = function (IntValue $event) {
```

Comment:
nit but this could be 
```suggestion
        $fn = function (IntValue $event): IntValue {
```